packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -219,15 +219,12 @@ export function declineIncomingCall(incomingCall) {
   };
 }
 
-export function acceptIncomingCall(incomingCall, spark) {
+export function acceptIncomingCall(incomingCall) {
   return (dispatch) => {
-    spark.phone.createLocalMediaStream()
-      .then((localMediaStream) => {
-        dispatch(storeCall(incomingCall));
-        incomingCall.answer({localMediaStream})
-          .then(() => {
-            dispatch(removeCallIncoming(incomingCall));
-          });
+    dispatch(storeCall(incomingCall));
```

Comment:
went ahead and wiped out the local media stream stuff that is handled by the sdk